app/backend/aquifers/migrations/0038_aquifer_resource_section_update.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.28 on 2022-11-29 02:22
+
+from django.db import migrations
+
+UPDATE_AQUIFER_RESOURCE_SECTION = """
+    UPDATE aquifer_resource_section_code 
+    SET name = 'Drilling and operation advisory'
+    WHERE name = 'Artesian advisory';
+"""
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aquifers', '0037_auto_20210224_2343'),
+    ]
+
+    operations = [
+        migrations.RunSQL(UPDATE_AQUIFER_RESOURCE_SECTION)
```

Comment:
I think we should provide a `reverse_sql` here, just in case we need to rollback the migration.

Reply:
Added reverse migration sql code